<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlLightEditor</class>
 <widget class="QGroupBox" name="GlLightEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string> </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="number_label">
     <property name="text">
      <string>Light Number</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="number_spinBox">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="type_comboBox">
     <item>
      <property name="text">
       <string>Directional</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Positional</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="colors_gridLayout">
     <item row="0" column="0">
      <widget class="QtColorPicker" name="ambient_colorpicker">
       <property name="text">
        <string>Ambient</string>
       </property>
       <property name="useStandardColors" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QtColorPicker" name="diffuse_colorpicker">
       <property name="text">
        <string>Diffuse</string>
       </property>
       <property name="useStandardColors" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QtColorPicker" name="specular_colorpicker">
       <property name="text">
        <string>Specular</string>
       </property>
       <property name="useStandardColors" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QPushButton</extends>
   <header>qtcolorpicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
